tests/e2e/klickerKnight.test.ts

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
+import { execSync } from 'child_process';
+import { databaseUtil } from '../../src/utils/databaseUtil';
+
+describe('klicker-knight', () => {
+  after(() => {
+    databaseUtil.delete();
+  });
+
+  it('trips on a flagstone', () => {
+    const result = execSync('npm run --slient klicker-knight');
```

Comment:
How is the `--slient` flag being used here? Would this have a output?

Reply:
It's an `npm` flag that suppresses the extra output from npm like 

```bash
> klicker-knight@0.0.0 klicker-knight
> ts-node src/klickerKnight

# actual output goes here
```

If you check the "Test E2E" output in the [ci](https://github.com/normal-devs/klicker-knight/runs/5014633249?check_suite_focus=true) for this branch you can see the assertion runs and passes, which can only happen if it returns the output.

Reply:
This is what I see
```
➜  klicker-knight git:(remove-oclif) ✗ npm run klicker-knight

> klicker-knight@0.0.0 klicker-knight /home/john/src/klicker-knight
> ts-node src/klickerKnight


You helplessly stare into the void as you try to gain your bearings. Your foot catches on what you believe to be a loose flagstone. You fall and break your neck. You are dead.%
➜  klicker-knight git:(remove-oclif) ✗ npm run --slient klicker-knight

> klicker-knight@0.0.0 klicker-knight /home/john/src/klicker-knight
> ts-node src/klickerKnight


You helplessly stare into the void as you try to gain your bearings. Your foot catches on what you believe to be a loose flagstone. You fall and break your neck. You are dead.% 
➜  klicker-knight git:(remove-oclif) ✗
```

I don't see anything extra without the `--slient` flag. Also do you have a `%` at the end of your output?

Reply:
1) Welp that's because I put `slient` instead of `silent`. Classic.

2) When I use zsh it puts a `%` at the end of some commands. Looks like [this is the answer](https://unix.stackexchange.com/questions/167582/why-zsh-ends-a-line-with-a-highlighted-percent-symbol)

Reply:
Fixed the typo. The `--silent` isn't necessary, but if you're debugging it you'd get the output that we care about.

Reply:
In production you wouldn't use npm to run the game, but we don't have a production

Reply:
Ah. Didn't see the typo. It seems to be working now.